pulser-core/pulser/sequence.py

```diff
@@ -846,7 +846,7 @@ def target(
     def target_index(
         self,
         qubits: Union[int, Iterable[int], Parametrized],
-        channel: Union[str, Parametrized],
+        channel: str,
```

Comment:
Sorry I missed this... 
Also, I think you also need to remove the `Parametrized` option on the regular `target`

Reply:
Do you mean on the "channel" argument? I think it's already away.

Reply:
I meant the `qubits` on `target()`, isn't it still allowing `Parametrized`?

Reply:
Do you think we should forbid it? Couldn't it be useful when the QubitId is an integer?

Reply:
I think there should be only one way to use variables on a target operation, and that's `target_index`. I would prefer that the `target()` operation stops supporting parametrized objects as arguments now that `target_index` exists.

Reply:
Ok. Then, I guess I should also remove the `Parametrized` possibility of the `initial_target` of `declare_channel`, to keep consistency.

Reply:
Indeed, please do!